sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelStatePartTest.java

```diff
@@ -162,6 +162,20 @@ void testRequirement54710() throws NoTestData {
         testRequirement547(metricCategory, activationState);
     }
 
+    @Test
+    @DisplayName("If pm:AbstractMetricDescriptor/@MetricCategory = Clc and the calculation for the METRIC is being"
+        + " performed, the SERVICE PROVIDER SHALL set pm:AbstractMetricState/@ActivationState = On.")
+    @TestIdentifier(EnabledTestConfig.BICEPS_547_12_0)
+    @TestDescription("For each metric with the category CLC, the device is manipulated to perform calculations and"
+        + " then it is verified that the ActivationState of the metric is set to On.")
+    @RequirePrecondition(manipulationPreconditions =
+        {ManipulationPreconditions.MetricStatusManipulationCLCActivationStateON.class})
+    void testRequirement547120() throws NoTestData {
+        final var metricCategory = MetricCategory.CLC;
+        final var activationState = ComponentActivation.ON;
+        testRequirement547(metricCategory, activationState);
```

Comment:
no handling of `AbstractMetric`s transmitted through WaveformStream. This is not likely for Clc and Set, but for Msrmt it is not unlikely.